demo/api/src/main.ts

```diff
@@ -39,6 +41,18 @@ async function bootstrap(): Promise<void> {
         }),
     );
 
+    app.use(
+        helmet({
+            contentSecurityPolicy: {
+                useDefaults: false,
+                directives: {
+                    "default-src": helmet.contentSecurityPolicy.dangerouslyDisableDefaultSrc,
+                    // locally: allow localhost in frame-ancestors to enable including files from the API in iframes in admin
+                    "frame-ancestors": `'self' ${process.env.NODE_ENV === "development" ? config.adminUrl : ""}`,
```

Comment:
This has the disadvantage that it's not defined in the library. So it doesn't work out-of-the-box in all our projects

Reply:
Does it work when set to `'self'` instead of the admin url or localhost:* ?

Reply:
No it doesn't work with only `'self'` because api and admin are not running under the same host. I tested it